docs/dev/release.rst

```diff
@@ -188,20 +183,78 @@ You can create a clean checkout as follows (requires
      Other Changes and Additions
      ---------------------------
 
-#. Commit your changes of the, uh, change log::
+     A.C.1 (unreleased)
+     ==================
 
-     git add CHANGES.rst
-     git commit -m "Back to development: A.B.dev"
+     Bug Fixes
+     ---------
 
-#. Push out the updated code and tag. If applicable, change ``origin`` to point to
-   the remote that points to the repository being released::
+#. Commit your changes of the, uh, change log with a message, "Back to development: A.B.dev"
 
-     git push origin main
-     git push origin vX.Y.Z
+#. Follow procedures for :ref:`release-milestones`.
+
+#. For your own sanity unrelated to the release, grab the new tag for your fork::
+
+     git fetch upstream --tags
+
+Congratulations, you have just released a new version of Jdaviz!
+
+.. _release-bugfix:
+
+Releasing a bugfix version
+==========================
+
+The procedure for a bugfix release is a little different from a feature release - you will
+be cherry-picking bugfixes into an existing release branch, and will also need to do some
+cleanup on the ``main`` branch.
+
+#. Lock down the ``main`` branch of the repository by setting the
+   `branch protection <https://github.com/spacetelescope/jdaviz/settings/branches>`_
+   rule for ``main`` to some high number required to merge, so that more PRs don't
+   get merged while you're releasing.
```

Comment:
this might actually not be necessary - but doesn't hurt - since you aren't pushing anything to main (except I suppose the change log update at the end)

Reply:
Doesn't hurt, especially when there are multiple people with write access. As team grows/changes, sometimes there can be communication breakdown.

Reply:
This isn't strictly necessary, but as you said makes sure the change log doesn't change on main in the middle of releasing.